### DISCUSSION TITLE
How to convert a string to url encoded format?

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2585] [Q&A]
Hi, I little struck with converting a string to url encoded format. can anyone please suggest me the best approach?

Example: I have string like the below and have special characters.
ItemID= AAMkADIwOTliZTRhLTg1NWUtNDM3Yi1iNGRkLTE0MGIwYzdhOWY1NABGAAAAAACAnTfH9ukGSrmKTWQM2kvvBwCLtIfIZ8waSZPeJewEgnwlAAAAAAEMAACLtIfIZ8waSZPeJewEgnwlAAD
FHr1**-**AAA**=** 

So the above url is in handlebar like {{ItemID}}

would like to replace the special characters like "-.&,= etc" to url encoded supported format?

**Comment by wobba** [comment]
See https://github.com/helpers/handlebars-helpers#replace
